tests/unit/test_configuration.py

```diff
@@ -1,21 +1,44 @@
+from pathlib import Path
+
 import pytest
 from vivarium.config_tree import ConfigTree
 
 from pseudopeople.entities import Form
-from pseudopeople.utilities import get_default_configuration
+from pseudopeople.utilities import (
+    get_default_configuration,
+    update_configuration_with_yaml,
+)
+
+
+@pytest.fixture
+def user_configuration_yaml(tmp_path):
+    text = """decennial_census:
```

Comment:
Consider using the `yaml` library and the `yaml.dump()` function rather than writing raw strings to file. With `yaml.dump()` you can write a nested dict directly to yaml. I don't feel strongly one way or another in this case though.

```
config = {
    "decennial_census": {
        "omission": 0.05,
        "first_name": {
            "nickname": {
                "row_noise_level": 0.05
            }
        }
    }
}
with open('output.yaml', 'w') as file:
    yaml.dump(config, file)
```

Reply:
It's certainly cleaner to use a dictionary literal than a multi-line string of YAML where the whitespace is a little unclear.